CMakeLists.txt

```diff
@@ -552,8 +551,6 @@ if (NOT EVENT__DISABLE_THREAD_SUPPORT)
     endif()
 endif()
 
-# Add stdio.h for vasprintf
-list(APPEND CMAKE_EXTRA_INCLUDE_FILES ${EVENT_INCLUDES} stdio.h)
```

Comment:
This is required for gettimeofday (and other functions) detection

Reply:
I can't push to your topic branch (seems that you forbid this), so please fix this by yourself (with rebase against upstream/master)

Reply:
You can see the failure here - https://github.com/libevent/libevent/pull/1147/checks?check_run_id=2210835157

Other builds does not fails because it reuses CMakeCache.txt

Reply:
No problem. I've addressed to just remove the comment given it is incorrect.